perf/lua/box_select.lua

```diff
@@ -222,6 +229,15 @@ local TESTS = {
     },
 }
 
+local function single_run(func)
+    local real_time_start = clock.time()
+    local cpu_time_start = clock.proc()
+    func()
+    local delta_real = clock.time() - real_time_start
+    local delta_cpu = clock.proc() - cpu_time_start
+    return delta_real, delta_cpu
+end
```

Comment:
I see that a similar helper is added to several benchmarks. How about providing it directly from the `benchmark` module?

Reply:
I'm not sure about its usage compared to the other benchmarks, since for this particular benchmark we want to just sum all timings, but this may not be suitable for others. So, I prefer to postpone this decision until the moment when we will have some common pattern for writing such benchmarks. Maybe this functionality should be implemented via the `--benchmark_repetitions` option.